app/helpers/date_helper.rb

```diff
@@ -3,8 +3,8 @@ def format_date_of_birth(date)
     I18n.l(date.to_date, format: :date_of_birth)
   end
 
-  def format_date_of_visit(date)
-    I18n.l(date.to_date, format: :date_of_visit)
+  def format_near_date(date)
```

Comment:
I find 'near date' somewhat confusing in contrast to 'date of visit'.  It refers to a date in the near future?